{"name": "Ручка /api/v1/orders/accept/", "status": "failed", "statusDetails": {"message": "AssertionError: assert 404 == 400\n +  where 404 = <Response [404]>.status_code", "trace": "self = <test_api.tests.test_accept_order.TestAcceptOrder object at 0x0000027A8CBEB3A0>, track_number = '', id_courier = ''\n\n    @allure.title(\"Ручка /api/v1/orders/accept/\")\n    @allure.description(\n        \"Принимаем заказ с комбинациями заполненности полей: трек номер и курьер\")\n    @pytest.mark.parametrize(\n        \"track_number, id_courier\",\n        [\n            AcceptOrder.Accept_0,\n            AcceptOrder.Accept_1,\n            AcceptOrder.Accept_2\n        ]\n    )\n    def test_accept_order_without_courier_id_or_track_number_error(self, track_number, id_courier):\n        with allure.step('Запрос на принятие заказа по трек-номеру и некорректному ID курьера'):\n            response = requests.put(URLS.URL_MAIN + URLS.URL_ACCEPT_ORDER + str(track_number) + \"?\" + \"courierId=\" + str(id_courier))\n    \n        with allure.step('Проверки на статус и ответ'):\n>           assert response.status_code == 400\nE           assert 404 == 400\nE            +  where 404 = <Response [404]>.status_code\n\ntest_api\\tests\\test_accept_order.py:69: AssertionError"}, "description": "Принимаем заказ с комбинациями заполненности полей: трек номер и курьер", "steps": [{"name": "Запрос на принятие заказа по трек-номеру и некорректному ID курьера", "status": "passed", "start": 1710620269096, "stop": 1710620269145}, {"name": "Проверки на статус и ответ", "status": "failed", "statusDetails": {"message": "AssertionError: assert 404 == 400\n +  where 404 = <Response [404]>.status_code\n", "trace": "  File \"C:\\!Learning\\Yandex_auto\\qa_sprint_7\\test_api\\tests\\test_accept_order.py\", line 69, in test_accept_order_without_courier_id_or_track_number_error\n    assert response.status_code == 400\n"}, "start": 1710620269145, "stop": 1710620269145}], "parameters": [{"name": "track_number", "value": "''"}, {"name": "id_courier", "value": "''"}], "start": 1710620269096, "stop": 1710620269145, "uuid": "eaed4199-eeb6-4777-b201-17050717a278", "historyId": "6276f43c1f2f011d0dd42e5d7baf5b83", "testCaseId": "6276f43c1f2f011d0dd42e5d7baf5b83", "fullName": "test_api.tests.test_accept_order.TestAcceptOrder#test_accept_order_without_courier_id_or_track_number_error", "labels": [{"name": "parentSuite", "value": "test_api.tests"}, {"name": "suite", "value": "test_accept_order"}, {"name": "subSuite", "value": "TestAcceptOrder"}, {"name": "host", "value": "LAPTOP-KS9V29SF"}, {"name": "thread", "value": "23412-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_api.tests.test_accept_order"}]}